<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="anslabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>1291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>20</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать вектор</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>384</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="ErasepushButton">
         <property name="text">
          <string>erase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RevercepushButton_2">
         <property name="text">
          <string>reserce</string>
         </property>
        </widget>
     
  </item>
       <item>
        <widget class="QPushButton" name="AtpushButton_3">
         <property name="text">
          <string>at</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="max_sizepushButton_5">
         <property name="text">
          <string>max_size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ResizepushButton_6">
         <property name="text">
          <string>resize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearpushButton_7">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pop_backpushButton_9">
         <property name="text">
          <string>pop_back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sizepushButton_10">
         <property name="text">
          <string>size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="datapushButton_12">
         <property name="text">
          <string>data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="push_backpushButton_13">
         <property name="text">
          <string>push_back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="capacitypushButton_14">
         <property name="text">
          <string>capacyty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="emptypushButton_15">
         <property name="text">
          <string>empty</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>203</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Длинна начального вектора</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SizeSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Элемент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ElementSpinBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1363</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
